scripts/rollup/modules.js

```diff
@@ -52,6 +52,17 @@ const devOnlyFilesToStubOut = [
   "'ReactTestUtils'",
 ];
 
+// Ordering of these imports is important;
+// The default import must follow deep imports or Rollup breaks.
```

Comment:
Doesn't this mean that the replacement logic misses `'` or something that delimits end of the string, and we should add that?